Formula/lowdown.rb

```diff
@@ -0,0 +1,25 @@
+class Lowdown < Formula
+  desc "Markdown translator with a troff(1) backend"
+  homepage "https://kristaps.bsd.lv/lowdown/"
+  url "https://kristaps.bsd.lv/lowdown/snapshots/lowdown-0.1.9.tar.gz"
+  sha256 "3b809f9cb4cf6912bb2f70666cdf186d2553f1abfd549ab537b782b4688c4e71"
+
+  def install
+    localconfig = [
+      # Sane prefixes.
+      "PREFIX=#{prefix}",
+      "INCLUDEDIR=#{include}",
+      "BINDIR=#{bin}",
+      "LIBDIR=#{lib}",
+      "MANDIR=#{man}",
+    ]
+    (buildpath/"configure.local").write localconfig.join("\n")
```

Comment:
The `mandoc.rb` formula uses an array for the initial values; then append append some depending on the options it’s build with; then join everything together. This is a common pattern in formulae with options; but here it’s not necessary.